tools/DynamoPackagesAnalyzer/Extensions/DirectoryInfoExtensions.cs

```diff
@@ -0,0 +1,22 @@
+namespace System.IO
+{
+    /// <summary>
+    /// Provides extensions method for DirectoryInfo
+    /// </summary>
+    internal static class DirectoryInfoExtensions
```

Comment:
It may work, but I think that would be easier for developers that may extend the project in the future to understand how it works if we create an extension class per class, with this approach we are able to use the same namespace of the class we are extending, and use the extension without the need of importing additional namespaces, just the extended class namespace